docs/developer/serving/services/custom-domains.md

```diff
@@ -7,6 +7,10 @@ You can create a `DomainMapping` object to map a single, non-wildcard domain to
 For example, if you own the domain name `example.org`, and you configure the domain DNS to reference your Knative cluster, you can use domain mapping to
 serve a Knative Service at this domain.
 
+!!! note
+    If you create a domain mapping to map to a [private Knative Service](../developer/serving/services/private-services.md),
+    the private Knative Service will be accessible from public internet with the custom domain of the domain mapping.
```

Comment:
```suggestion
!!! important
    If you create a `DomainMapping` object that maps to a [private Knative Service](../private-services), the private Service will be accessible from public internet by accessing the custom domain.
```

Not sure if the link I'm suggesting is correct, however since this file is already in the `../developer/serving/services/` directory, I think you can use a relative link here?

Reply:
Good catch. done